### DISCUSSION TITLE
Epic Fight Mod / Leawind's Third Person Compatibility

### DISCUSSION THREAD

**Discussion by Smoker120900** [Leawind/Third-Person#84] [Ideas]
Hello, I'm here representing the team behind Epic Fight Mod
I wanted to ask if you could lend us a hand to make both mods 100% compatible with each other, something we are strugling to because of some camera issues.
This is our Discord, please notify us of your Username and when you get around the server so we could more easily contact with you:
https://discord.gg/epic-fight-955784714264932362
Best regards, Smoker from the Epic Fight Mod team

**Comment by Leawind** [comment]
Hello! Sorry for replying so late. My Discord username is Leawind.
I'm willing to help solve the compatibility issue, but I've been quite busy lately and haven't had much time to develop the mod. Maybe I'll be free two months later.
Thanks for the invitation and support!

**Reply by AxelEncore** [reply]
Thank you, we are all really looking forward to it
